main.go

```diff
@@ -108,36 +108,41 @@ func main() {
 					return
 				}
 
-				// validate the cluster ids contained in the remote write request
-				clusterId, err := remotewrite.ValidateRequest(remoteWriteRequest)
-				if err != nil {
-					log.Printf("error validating remote write request: %v", err)
-					w.WriteHeader(http.StatusForbidden)
-					return
-				}
+				clusterId := ""
```

Comment:
in Go acronyms should be uppercased as long as exportability is not affected for the variable. In this case
it should be `clusterID := ==` or directly not needing to explicitly initialize it:
`var clusterID string`